miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/SampleAdditionalInfoDto.java

```diff
@@ -30,6 +33,8 @@
   private String labUrl;
   private Long groupId;
   private String groupDescription;
+  private Long updatedById;
+  private String updatedByUrl;
```

Comment:
SampleDto could have a lastModifier field, but this updatedBy has been removed from SampleAdditionalInfo. The Dtos.asDetailedSample(SampleAdditionalInfo) method above is actually pulling the value from Sample, not SampleAdditionalInfo